### DISCUSSION TITLE
Failing to transmit from Docker via GST SRT Sink

### DISCUSSION THREAD

**Discussion by MuradMY** [Haivision/srt#2129] [Q&A]
I am trying to transmit from a docker image (which has GST SRT Sink) to host (my mac) and catching it, but it is failing.

This is the command I use:
`gst-launch-1.0 -v videotestsrc pattern=snow ! video/x-raw, height=1080, width=1920 ! videoconvert ! x264enc tune=zerolatency ! video/x-h264, profile=high ! mpegtsmux ! srtsink uri="srt://host.docker.internal:8888"`

And this is the result:
`ERROR: from element /GstPipeline:pipeline0/GstSRTSink:srtsink0: Could not write to resource.
Additional debug info:
../subprojects/gst-plugins-bad/ext/srt/gstsrtobject.c(1436): gst_srt_object_send_headers (): /GstPipeline:pipeline0/GstSRTSink:srtsink0:
Connection does not exist
ERROR: from element /GstPipeline:pipeline0/GstSRTSink:srtsink0: Failed to write to SRT socket: Unknown error
Additional debug info:
../subprojects/gst-plugins-bad/ext/srt/gstsrtsink.c(181): gst_srt_sink_render (): /GstPipeline:pipeline0/GstSRTSink:srtsink0`

Any idea what I could be doing wrong?

**Comment by maxsharabayko** [comment]
Looks like a connectivity issue? Check the server is reachable first, e.g. with `srt-live-transmit`.
